### ISSUE TITLE
fosdem with UID 1000 might conflict with pre-existing users

### ISSUE THREAD

**Issue by yoe** [FOSDEM/infrastructure#193]
scaleway defaults to creating a `debian` user on their Debian images.

These ansible scripts also want to create a `fosdem` user, on the same uid of 1000. This creates an error situation:

We should do one of the following:

- Remove the "debian" user with uid 1000 if it exists
- Don't hardcode the uid, allow ansible to create one (though this might result in problems with NFS or similar protocols that want to encode UIDs on the network)
- Switch the UID to something else; 1000 is very likely to already be in use by something else.

**Comment by RichiH** [comment]
Yes, it makes sense to start all our stuff at 10000 or something.

**Comment by gerryd** [comment]
I can live with that, but I would also delete the default `debian` user account with the scaleway role. There's no reason for it to exist...

**Comment by SuperQ** [comment]
+1 to both 10,000 UID range and removing the `debian` user.

I was also thinking about moving us away from direct root login and default to `fosdem` user + sudo. I can open a separate issue to discuss that.
